src/screens/OtherProfile/index.js

```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
 import SimpleToast from 'react-native-simple-toast';
 import ToastMessage from 'react-native-toast-message';
+import netInfo from '@react-native-community/netinfo';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `netInfo` from `@react-native-community/netinfo` has been added. Ensure that this package is properly installed and used in the codebase, as it's not visible in the provided hunks.
```diff
+ import netInfo from '@react-native-community/netinfo';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->